src/sketchup-stl/exporter.rb

```diff
@@ -230,6 +225,27 @@ def self.get_vertex_order(positions, face_normal)
         order
       end
 
+      # Return model.active_entites, selection, or nil
+      def self.get_export_entities
+        export_ents = nil
+        if OPTIONS['selection_only']
+          if Sketchup.active_model.selection.length > 0
+            export_ents = Sketchup.active_model.selection
+          else
+            msg = "SketchUp STL Exporter:\n\n"
+            msg << "You have chosen \"Export only current selection\", but nothing is selected."
+            msg << "\n\nWould you like to export the entire model?"
+            if UI.messagebox(msg, MB_YESNO) == IDYES
+              export_ents = Sketchup.active_model.active_entities
+            end
+          end
+        else
+          export_ents = Sketchup.active_model.active_entities
+        end
+        return export_ents
```

Comment:
Style guide says to not use `return` unless returning early - instead rely on the implicit return.